stable/voyager/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Voyager provides controller for Ingress and Certificates for Kubernetes developed by AppsCode.
 icon: https://cdn.appscode.com/images/icon/voyager.png
 name: voyager
-version: 0.1.0
+version: 0.2.0
```

Comment:
@unguiculus , Actually I would some advice on how to handle chart version-ing w.r.t actual application versioning? My main intention is to avoid confusion for users in terms of Chart version vs application (voyager) version. Is there any best practice in this regard?